models/articles.model.js

```diff
@@ -14,13 +13,35 @@ exports.selectArticleById = (article_id) => {
 };
 
 exports.selectArticles = () => {
-    return db.query(`
+  return db
+    .query(
+      `
       SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) ::INT AS comment_count 
       FROM articles 
       LEFT JOIN comments ON comments.article_id = articles.article_id
       GROUP BY articles.article_id
       ORDER BY articles.created_at DESC;
-    `).then(({ rows }) => {
+    `
+    )
+    .then(({ rows }) => {
       return rows;
     });
-  };
\ No newline at end of file
+};
+
+exports.updateArticle = (article_id, inc_votes) => {
+  return db
+    .query(
+      `UPDATE articles
+    SET votes = votes + $1
+    WHERE article_id = $2
+    RETURNING *`,
+      [inc_votes, article_id]
+    )
+    .then((result) => {
+      const updatedArticle = result.rows[0];
+      if (updatedArticle.votes < 0) {
+        updatedArticle.votes = 0;
+      }
+      return updatedArticle;
```

Comment:
Cool, here you will want to check the length of the RETURNING rows, if its 0, throw a article doesn't exist (e.g.999) error :) 

Your other 2 error tests should be handled by SQL error codes :)